CONTRIBUTING.md

```diff
@@ -2,11 +2,13 @@
 
 Thank your for contributing to this project! We welcome collaborators and expect users to follow our [code of conduct](CODE_OF_CONDUCT.md) when submitting code or comments.
 
+Note: The base branch requires [all commits to be signed](https://help.github.com/articles/about-commit-signature-verification). 
+
 1. Fork the repo (https://github.com/mana-ethereum/mana/fork).
 2. Create your feature branch (`git checkout -b my-new-feature`).
 3. Write tests that cover your work.
 4. Ensure CI rules are satisfied. 
-5. Commit your changes (`git commit -am 'Add some feature'`).
+5. Commit your changes (`git commit -S -am 'Add some feature'`).
```

Comment:
I know this was here before but what is the `am` flag for?

Reply:
Missing dash for -m flag, will remedy.

Reply:
Oh of course. So this could be `git commit -Sam 'Add some feature'`

Reply:
But I agree that keep the flags separate is clearer 👍